packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1460,6 +1460,7 @@ class StackViewport extends Viewport implements IStackViewport {
     };
 
     triggerEvent(eventTarget, Events.STACK_VIEWPORT_NEW_STACK, eventDetail);
+    this.viewportStatus = ViewportStatus.LOADING;
 
```

Comment:
This is still odd to me, I have not seen anywhere a LOADED event, and I"m pretty sure this is not loading, since the promise above is awaited